Assets/Scripts/UI/Utils/InputDefaultValueOnEnd.cs

```diff
@@ -0,0 +1,21 @@
+using UnityEngine;
+using UnityEngine.UI;
+
+[RequireComponent(typeof(InputField))]
+public class InputDefaultValueOnEnd : MonoBehaviour
```

Comment:
How is this actually used?

Reply:
This component is added to any object with an input field. The "RequireComponent" tag will enforce this requirement. It functions automatically. Whatever you enter (in the editor) for the "DefaultValue" is automatically injected into the input field if the user deselects the input field and leaves it empty.